.github/workflows/ci-fortuna.yml

```diff
@@ -0,0 +1,20 @@
+name: Check Fortuna
+
+on:
+  pull_request:
+    paths: [fortuna/**]
+  push:
+    branches: [main]
+    paths: [fortuna/**]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly-2023-07-23
+          override: true
+      - name: Run executor tests
+        run: cargo test --manifest-path ./fortuna/Cargo.toml
```

Comment:
there aren't any unit tests right now, but this checks the build and will run the tests once we add some